packages/components/details/_details.scss

```diff
@@ -137,24 +137,38 @@
     background-size: 32px;
     color: $color_nhsuk-blue;
     cursor: pointer;
-    display: block;
+    display: inline-block;
     padding: nhsuk-spacing(2) nhsuk-spacing(2) nhsuk-spacing(2) 38px;
     position: relative;
   }
 
   .nhsuk-details__text {
     @include nhsuk-responsive-padding(4);
+
     border-left: 0;
     margin-left: 0;
     margin-top: 0;
+    padding-top: 0 !important;
```

Comment:
Use of `!important` here, can this be avoided? If not can we add an eslint disable to this line and a comment as to why.

Reply:
Refactored `padding-top: 0 !important;`